app/views/selfservicetimetopay/arrangement/application_complete.scala.html

```diff
@@ -31,21 +29,23 @@
   sidebarLinks = Some(selfservicetimetopay.partials.help_and_advice_sidebar()),
   googleAnalyticsCalls = Seq(gaCheckoutCode.body)
 ){
-  <div class="transaction-banner--complete">
+  <div class="transaction-banner--complete removeBottom">
     <div class="transaction-banner__heading" role="banner">@Messages("ssttp.arrangement.complete.title")</div>
-    <p class="heading-large" id="ssttpOutcome">@Messages("ssttp.arrangement.complete.subtitle")</p>
+      <p>@Messages("ssttp.arrangement.complete.body")</p>
   </div>
   <section class="section divider--bottom">
-    <h2>@Messages("ssttp.arrangement.complete.next")</h2>
+          <p><a href="javascript:window.print()" class="js-visible no-print float--right  print-link js-visible trackedPrintLink openOnPrint">@Messages("ssttp.arrangement.instalment-summary.print")</a></p>
+          <h2>@Messages("ssttp.arrangement.complete.next")</h2>
     <p>@Messages("ssttp.arrangement.complete.summary")</p>
       <button onclick="window.print()" class="js-visible no-print button--secondary js-visible trackedPrintLink openOnPrint">@Messages("ssttp.arrangement.complete.print")</button>
```

Comment:
Do we need the window.print()? The button just needs to be a dropdown to display the hidden data so this might not be necessary